src/examples/mnist/MNIST_SP.cpp

```diff
@@ -157,7 +157,7 @@ void test() {
       /* infer */           true,
                             result);
     // Check results
-    const auto cls = result.getClass();
+    UInt cls = max_element(result[0].begin(), result[0].end()) - result[0].begin();
```

Comment:
can you explain this a bit more? (what is in result[i], what are values result[0].* ?) 
Or better yet, make a method `Classifier::getResult` (that either takes result arg, or uses classifier's member `Result current_;`

Reply:
I added a new function `getClassification(const PDF &)->UInt`